content/id/docs/concepts/extend-kubernetes/extend-cluster.md

```diff
@@ -0,0 +1,188 @@
+---
+title: Memperluas Kubernetes Cluster Anda
+content_template: templates/concept
+weight: 10
+---
+
+{{% capture overview %}}
+
+Kubernetes sangat mudah dikonfigurasi dan dikembangkan. Sehingga,
+jarang membutuhkan fork atau menambahkan patches ke kode projek Kubernetes.
+
+Panduan ini menjelaskan pilihan untuk menyesuaikan Kubernetes cluster.
+Ini ditujukan kepada {{< glossary_tooltip text="cluster operators" term_id="cluster-operator" >}} yang ingin
+memahami bagaimana menyesuaikan kluster Kubernetes dengan kebutuhan lingkungan kerja mereka.
+Pengembang yang prospektif {{< glossary_tooltip text="Platform Developers" term_id="platform-developer" >}} atau {{< glossary_tooltip text="Contributors" term_id="contributor" >}} Projek Kubernetes juga mendapatkan manfaat dari
+dokumen ini sebagai pengantar apa saja poin dan pola yang ada, untung-rugi dan batasannya.
+
+{{% /capture %}}
+
+
+{{% capture body %}}
+
+## Gambaran
+
+Pendekatan kostumisasi secara luas dapat dibagi atas *konfigurasi*, yang hanya melibatkan perubahan bendera, konfigurasi file lokal, atau API sumber;
```

Comment:
```suggestion
Pendekatan-pendekatan kostumisasi secara umum dapat dibagi atas _konfigurasi_, yang hanya melibatkan perubahan _flag_, konfigurasi berkas lokal, atau objek-objek sumber daya API;
```

API resources here refers to API resources objects. I think we can translate it into "objek-objek sumber daya API"

Reply:
Also, we use underscores to italicize words.
We use double asterisks to bold words.
This is to standardize our formats

Reply:
I think translating _flag_ into bendera here is a bit bizzare for the readers to read, as that's not a common term for technology (bendera) in Indonesia.
I think it's better to keep it as is and italicize it

Reply:
done